### ISSUE TITLE
Unexpected whitespace inserted in Wakati mode

### ISSUE THREAD

**Issue by himkt** [buruzaemon/natto-py#114]
I suspect that this bug is introduced in #111, sorry...

```python
import natto

print(natto.__version__)
nm = natto.MeCab('-Owakati')
print(nm.parse('この星の１等賞になりたいの、俺はっ！')[-1])
```

```
0.9.1
 # <- whitespace!!
```

**Comment by buruzaemon** [comment]
Ah, I apologize! Looking into this.

**Comment by buruzaemon** [comment]
This will be soon resolved with the 0.9.2 release. Happy New Year!
